stabilize.md

```diff
@@ -44,44 +47,162 @@ Flap Semantics
 --------------
 
 ```k
-    syntax Bid ::= Bid ( Int, Int, Address, Int, Int ) [klabel(BidBid)]
- // -------------------------------------------------------------------
+    syntax Bid ::= Bid ( bid: Int, lot: Int, guy: Address, tic: Int, end: Int ) [klabel(BidBid)]
+ // --------------------------------------------------------------------------------------------
 
     syntax MCDStep ::= "Flap" "." FlapStep
  // --------------------------------------
+    rule <k> step [ Flap . FAS:FlapAuthStep ] => Flap . push ~> Flap . auth ~> Flap . FAS ~> Flap . catch ... </k>
```

Comment:
can we not add these blocks? step, auth, rely, deny, push, pop, drop, exception, catch. I am removing them in my upcoming pr and it will make the merge easier if we don't.